<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SK-20210202NYIX" ignored="3" name="API_AutoFramework_Demo(failed)" tests="2" failures="2"
           timestamp="10 三月 2021 06:44:44 GMT" time="0.001" errors="0">
    <testcase name="test05" time="0.001" classname="com.qa.z_liuhaoqun_lenarn_canshu.ReRunFailedTestDemo">
        <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
            <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.z_liuhaoqun_lenarn_canshu.ReRunFailedTestDemo.test05(ReRunFailedTestDemo.java:34)
at com.qa.z_liuhaoqun_lenarn_canshu.a_first_ReRunTestNg.main(ReRunTestNg.java:30)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- test05 -->
    <testcase name="test03" time="0.0" classname="com.qa.z_liuhaoqun_lenarn_canshu.ReRunFailedTestDemo">
        <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
            <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.z_liuhaoqun_lenarn_canshu.ReRunFailedTestDemo.test03(ReRunFailedTestDemo.java:23)
at com.qa.z_liuhaoqun_lenarn_canshu.a_first_ReRunTestNg.main(ReRunTestNg.java:30)
... Removed 26 stack frames]]>
        </failure>
    </testcase> <!-- test03 -->
    <testcase name="test04" time="0.0" classname="com.qa.z_liuhaoqun_lenarn_canshu.ReRunFailedTestDemo">
        <ignored/>
    </testcase> <!-- test04 -->
    <testcase name="test02" time="0.0" classname="com.qa.z_liuhaoqun_lenarn_canshu.ReRunFailedTestDemo">
        <ignored/>
    </testcase> <!-- test02 -->
    <testcase name="test01" time="0.0" classname="com.qa.z_liuhaoqun_lenarn_canshu.ReRunFailedTestDemo">
        <ignored/>
    </testcase> <!-- test01 -->
</testsuite> <!-- API_AutoFramework_Demo(failed) -->
